gazebo/PKGBUILD

```diff
@@ -25,7 +25,7 @@ optdepends=('bullet: Bullet support'
             'libusb: USB peripherals support'
             'simbody: Simbody support'
             'urdfdom: Load URDF files')
-makedepends=('cmake' 'git' 'ninja' 'doxygen' 'ruby-ronn')
+makedepends=('cmake' 'git' 'make' 'doxygen' 'ruby-ronn')
```

Comment:
You don't need to list ``make`` as a dependency. Members of ``base-devel`` are implicitly assumed to be installed for AUR packages: https://wiki.archlinux.org/title/Arch_User_Repository#Prerequisites